documentation/concepts/!concept-template.md

```diff
@@ -72,8 +73,8 @@ Table: {Table title which concisely explains the comparison.}
 
 | Option   | Pros                                  | Cons                                  |
 |----------|---------------------------------------|---------------------------------------|
-| {Concept} option 1 | <ul><li>Pro 1 for Option 1</li><li>Pro 2 for Option 1</li></ul> | <ul><li>Con 1 for Option 1</li><li>Con 2 for Option 1</li></ul> |
-| {Concept} option 2 | <ul><li>Pro 1 for Option 2</li><li>Pro 2 for Option 2</li></ul> | <ul><li>Con 1 for Option 2</li><li>Con 2 for Option 2</li></ul> |
+| {Concept} option 1 | - Pro 1 for Option 1 <br> - Pro 2 for Option 1 | - Con 1 for Option 1 <br> - Con 2 for Option 1 |
```

Comment:
I didn't use HTML just for fun! Unfortunately, your simplification results in a non-bulleted list, just some ticks. So I'd say keep the old way.

[Before](https://developer.gov.bc.ca/docs/default/component/aps-infra-platform-docs/concepts/!concept-template/#comparison-of-thing-being-compared-optional):
![image](https://github.com/user-attachments/assets/18d8707c-f9f3-41dd-9672-cea8b26c04bf)

After:
![image](https://github.com/user-attachments/assets/6dacb91c-5e6d-4ab1-b2c4-a934212fbb41)